clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2ManifestProvider.java

```diff
@@ -73,6 +78,14 @@ public KubernetesV2Manifest getManifest(String account, String location, String
       return null;
     }
 
+    Function<KubernetesManifest, String> lastEventTimestamp = (m) -> (String) m.get("lastTimestamp");
```

Comment:
https://kubernetes.io/docs/reference/generated/federation/v1/definitions/#_v1_event lists this field as not required, so we should probably have a reasonable default.

Reply:
Good point - I'll fall back to the "firstTimestamp", and then to "N/A" which'll sort higher than any timestamp